executable.go

```diff
@@ -177,3 +188,36 @@ func executableDefaultRetryHandler(err error) bool {
 		return false
 	}
 }
+func SetMaxBackoffForExecute(max time.Duration) {
```

Comment:
This method should exist on each request, not in `executable.go`. So `*[Transaction|Query].Set[Min|Max]Backoff()` where the setters return the request.